provider/oci/storage_volumes.go

```diff
@@ -154,11 +163,21 @@ func (v *volumeSource) CreateVolumes(ctx envcontext.ProviderCallContext, params
 	if params == nil {
 		return []storage.CreateVolumesResult{}, nil
 	}
+	var credErr error
+
 	results := make([]storage.CreateVolumesResult, len(params))
 	instanceMap := map[instance.Id]*ociInstance{}
 	for i, volume := range params {
-		vol, err := v.createVolume(volume, instanceMap)
+		if credErr != nil {
+			results[i].Error = errors.Trace(credErr)
+			continue
+		}
+		vol, err := v.createVolume(ctx, volume, instanceMap)
 		if err != nil {
+			if isAuthFailure(err, ctx) {
```

Comment:
Don't need this wrapper AND the call to common.HandleCredentialError(err, ctx) below
MaybeHandleCredentialError does the same thing that HandleCredentialError does